downloads/swan-lake-release-notes/swan-lake-alpha/RELEASE_NOTE.md

```diff
@@ -0,0 +1,1558 @@
+        - [Changes to Object Type Inclusion with Qualifiers](#changes-to-object-type-inclusion-with-qualifiers)
+        - [Changes to Record Type Inclusion with Rest Descriptors](#changes-to-record-type-inclusion-with-rest-descriptors)
+        - [Improved Listener Declaration](#improved-listener-declaration)
+        - [Referring Lang Library Modules Without Using Quoted Identifiers](#referring-lang-library-modules-without-using-quoted-identifiers)
+        - [Improved Lang Library Methods](#improved-lang-library-methods)
+    - [Runtime](#runtime)
+        - [Configurable](#configurable)
+    - [Developer Tools](#developer-tools)
+        - [Language Server](#language-server)
+        - [Project API](#project-api)
+        - [Debugger](#debugger)
+        - [Test Framework](#test-framework)
+        - [Bindgen Tool](#bindgen-tool)
+        - [Maven Resolver](#maven-resolver)
+        - [Ballerina Shell REPL [EXPERIMENTAL]](#ballerina-shell-repl-experimental)
+        - [Documentation](#documentation)
+    - [Standard Library](#standard-library)
+        - [HTTP Module Improvements](#http-module-improvements)
+        - [WebSocket Module Improvements](#websocket-module-improvements)
+        - [gRPC Module Improvements](#grpc-module-improvements)
+        - [Security Improvements](#security-improvements)
+        - [GraphQL Module Improvements](#graphql-module-improvements)
+        - [Common Changes in Messaging Modules](#common-changes-in-messaging-modules)
+        - [Kafka Module Improvements](#kafka-module-improvements)
+        - [NATS Module Improvements](#nats-module-improvements)
+        - [NATS Streaming Module Improvements](#nats-streaming-module-improvements)
+        - [RabbitMQ Module Improvements](#rabbitmq-module-improvements)
+        - [Time Module Improvements](#time-module-improvements)
+        - [Rename System Module to OS](#rename-system-module-to-os)
+        - [Runtime Module Improvements](#runtime-module-improvements)
+        - [Email Module Improvements](#email-module-improvements)
+        - [WebSub Module Improvements](#websub-module-improvements)
+        - [Introduced New Modules](#introduced-new-modules)
+        - [Removed Modules](#removed-modules)
+    - [Code to Cloud](#code-to-cloud)
+    - [Observability](#observability)
+    - [Breaking Changes](#breaking-changes)
+        - [Language](#language)
+    - [Taint Analyzer Update](#taint-analyzer-update)
+
+#### Updating Ballerina
+
+You can use the update tool to update to Ballerina Swan Lake Alpha as follows.
+
+##### For Existing Users
+
+If you are already using Ballerina, you can directly update your distribution to the Swan Lake channel using the [Ballerina Update Tool](/swan-lake/learn/keeping-ballerina-up-to-date/). To do this, first, execute the command below to get the update tool updated to its latest version. 
+                        
+> `bal update`
```

Comment:
@hevayo @keizer619 @sameerajayasoma This is incorrect, Because existing users, the command is `ballerina`

Reply:
Fixed.